libs/core/src/aabb.cpp

```diff
@@ -1,7 +1,16 @@
 #include "aabb.h"
-
+#define _USE_MATH_DEFINES
+#include <cmath>
 #include "glm/ext.hpp"
 
+
+// For non-GNU compilers (e.g. MSVC).
+// Consider switching to std::numbers::pi once C++20 support
+// is available.
+#ifndef M_PI_2
+#define M_PI_2 1.57079632679489661923  /* π/2 */

```

Comment:
Instead of manually defining this constant, we use the glm versions everywhere. Looks like you found the only place where glm wasn't used yet. That should be `glm::half_pi<double>()` instead of `M_PI_2` :)